server/datastore/mysql/migrations/tables/20230503101418_AlterJobsAddNotBefore.go

```diff
@@ -0,0 +1,22 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230503101418, Down_20230503101418)
+}
+
+func Up_20230503101418(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+ALTER TABLE jobs ADD COLUMN not_before TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
What do you think about increasing the precision of the timestamps?

Reply:
In general or in this particular case? I don't see the second-level precision being an issue for this particular case.

Reply:
I'm mostly asking in the context of this story. Sounds like we're good there then. My experiences with the schedule package and also the nano queue have just left me a little battle-scarred :)